mlc_llm/core.py

```diff
@@ -150,6 +150,16 @@ class BuildArgs:
             "action": "store_true",
         },
     )
+    use_cuda_graph: bool = field(
```

Comment:
should we by default enable cudagraph? not super familiar with cudagraph compatibility, does it support all sm_xx? if so, i suppose enabling by default should be a good idea

Reply:
Enabling cuda graph does something weird when profiling - all kernels that put into the graph are removed from the profiling report. This is an example output of `evaluate.py` when enabling CUDA graph:
```
======================= Decoding Profiling =======================
Name                                                             Time (ms)   Count   Total time (ms)   Percentage (%)
fused_relax_nn_attention_cutlass1                                0.0161      28      0.4502            98.74
take_decode                                                      0.0029      1       0.0029            0.63
cast1                                                            0.0029      1       0.0029            0.63
Total time: 0.4559 ms
```
And similarly, nvprof also doesn't report the kernel timing information for CUDA graph. 

Moreover, enabling CUDA graph adds high initialization overhead on the first call, since the graph is constructed just in time. So for example if I enable cuda graph and run `mlc_chat_cli`, I might get lower tok / s.

And finally, CUDA graph needs to be enabled very carefully if people start doing things like LoRA where input param tensors can change on every inference.

So I think cuda graph should be disabled by default.